### ISSUE TITLE
Add redirect URL in login

### ISSUE THREAD

**Issue by RohitKumar-200** [Bookshlf-in/Website#144]
login page always redirects to the home page after successful login. It would be nice if we redirect user to the page from which he clicked the log-in button.

For example:
Go on any book details page. After that click on the login button, you will now be redirected to the login page. After successful login, you will be redirected to the home page. But you should be redirected to the same book details page which was opened earlier.

You can achieve this functionality by providing route path as "/login/:redirect" and when clicked on the login button it should redirect to "/login/\<current_url>". After successful login, redirect page to URL that was passed as \<current_url>. If the redirect URL is not present, then redirect the user to the home page.

**Comment by RohitKumar-200** [comment]
@DKumar2169 Yes, you can work on this. I have assigned it to you. Feel free to ask any doubt.

**Comment by DKumar2169** [comment]
okay Thanks
so can you specify what files to changes/access

**Comment by RohitKumar-200** [comment]
> so can you specify what files to changes/access

You will have to make changes in these files:
* [App.js](https://github.com/Bookshlf-in/Website/blob/main/src/components/App/App.js)
* [Login.js](https://github.com/Bookshlf-in/Website/blob/main/src/components/Login/Login.js)
* [Navbar.js](https://github.com/Bookshlf-in/Website/blob/main/src/components/Navbar/Navbar.js)
* [Sidenav.js](https://github.com/Bookshlf-in/Website/blob/main/src/components/Navbar/Sidenav.js)

I think this would be sufficient. You can explore more and figure out what more files are needed to be changed to implement this feature.

**Comment by yash-jain24** [comment]
can i work on this?
